README.md

```diff
@@ -23,396 +23,345 @@ Stable release versions are manually released from the main branch.
-//          `activationHeight` from the next version of the global parameters.
-//    Below, these values are hardcoded, but they should be retrieved from the
-//    Babylon system.
-const covenantPks: Buffer[] = covenant_pks.map((pk) => Buffer.from(pk, "hex"));
-const covenantThreshold: number = 3;
-const minUnbondingTime: number = 101;
-const magicBytes: Buffer = Buffer.from("62627434", "hex"); // "bbt4" tag
-// Optional field. Value coming from current global param activationHeight
-const lockHeight: number = 0;
-
-// 2. Define the user selected parameters of the staking contract:
-//    - `stakerPk: Buffer`: The public key without the coordinate of the
-//       staker.
-//    - `finalityProviders: Buffer[]`: A list of public keys without the
-//       coordinate corresponding to the finality providers. Currently,
-//       a delegation to only a single finality provider is allowed,
-//       so the list should contain only a single item.
-//    - `stakingDuration: number`: The staking period in BTC blocks.
-//    - `stakingAmount: number`: The amount to be staked in satoshis.
-//    - `unbondingTime: number`: The unbonding time. Should be `>=` the
-//      `minUnbondingTime`.
-
-const stakerPk: Buffer = btcWallet.publicKeyNoCoord();
-const finalityProviders: Buffer[] = [
-  Buffer.from(finalityProvider.btc_pk_hex, "hex"),
-];
-const stakingDuration: number = 144;
-const stakingAmount: number = 1000;
-const unbondingTime: number = minUnbondingTime;
-
-// 3. Define the parameters for the staking transaction that will contain the
-//    staking contract:
-//    - `inputUTXOs: UTXO[]`: The list of UTXOs that will be used as an input
-//       to fund the staking transaction.
-//    - `feeRate: number`: The fee per tx byte in satoshis.
-//    - `changeAddress: string`: BTC wallet change address, Taproot or Native
-//       Segwit.
-//    - `network: network to work with, either networks.testnet
-//       for BTC Testnet and BTC Signet, or networks.bitcoin for BTC Mainnet.
-
-// Each object in the inputUTXOs array represents a single UTXO with the following properties:
-// - txid: transaction ID, string
-// - vout: output index, number
-// - value: value of the UTXO, in satoshis, number
-// - scriptPubKey: script which provides the conditions that must be fulfilled for this UTXO to be spent, string
-const inputUTXOs = [
-  {
-    txid: "e472d65b0c9c1bac9ffe53708007e57ab830f1bf09af4bfbd17e780b641258fc",
-    vout: 2,
-    value: 9265692,
-    scriptPubKey: "0014505049839bc32f869590adc5650c584e17c917fc",
-  },
-];
-const feeRate: number = 18;
-const changeAddress: string = btcWallet.address;
-const network = networks.testnet;
-```
+We will be using the full list of staking parameters in this library, for detailed
+explanation of each fields, refer to this [documentation](https://github.com/babylonlabs-io/babylon/blob/main/docs/register-bitcoin-stake.md#32-babylon-chain-btc-staking-parameters)
```

Comment:
```suggestion
explanation of the parameters, refer to the
[staking parameters specification](https://github.com/babylonlabs-io/babylon/blob/main/docs/register-bitcoin-stake.md#32-babylon-chain-btc-staking-parameters).
```

Reply:
We should link users to this document earlier as a pre-requisite to understand the process of registering stake.